.github/workflows/changesets.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    if: ${{ github.repository_owner == 'cloudflare' }}
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
do we need this? this will cause all history for every branch and tag to be fetched instead of just the single latest commit.

Reply:
Mmm, probably not, I was doing that before for using SHA's and tags for various context in other workflows.